tests/test_sweet.py

```diff
@@ -61,6 +61,7 @@ def test_sweet_vetter():
 
     sweet_vetter = Sweet()
     res = sweet_vetter.run(tce, lc)
+    sweet_vetter.plot()
```

Comment:
Usually, a unit test in CI does not do any plotting.

If you want to test the plot result, you should use https://matplotlib.org/3.1.0/api/testing_api.html .

Reply:
I'll create an issue that we should create a test or two that covers the plots. Because the plotters didn't have tests, we developed sweet and we didn't even notice that the plotting wasn't working.  It will take me more time than I have now, but I'll create an issue and we can see when it reaches  high enough priority to create the tests.